old-tutorials-in-development/tutorials-in-development/R/dc-spatio-temporal-intro/05-geographic-vs-projected-CRS.Rmd

```diff
@@ -276,34 +299,25 @@ the earth.
 respectively.  The origin for NAD 27 is Meades Ranch in Kansas.
 * *ED50* -- European Datum 1950
 
-For more information, read
+For more information, check out
 *  <a href="http://help.arcgis.com/en/arcgisdesktop/10.0/help/index.html#/Datums/003r00000008000000/" target="_blank">ESRI's ArcGIS discussion of Datums.</a>
 *  <a href="https://www.nceas.ucsb.edu/~frazier/RSpatialGuides/OverviewCoordinateReferenceSystems.pdf" target="_blank">page 2 in M. Fraiser's CRS Overview.</a>
 
 
-> NOTE: All coordinate reference systems have a vertical and horizontal datum
+Note that all coordinate reference systems have a vertical and horizontal datum
```

Comment:
@donal-at-NEON: Note paragraph edited for clarity. This doesn’t quite fit as a "data note" in my opinion so I made it part of the regular text. Not sure if that’s the best way, though. Let me know if I need to change this.